core/domain/state_domain.py

```diff
@@ -3109,9 +3109,12 @@ def _get_all_translatable_content(self):
         for ca_dict in self.interaction.customization_args.values():
             subtitled_htmls = ca_dict.get_subtitled_html()
             for subtitled_html in subtitled_htmls:
-                if subtitled_html.html != '':
+                html_string = subtitled_html.html
+                # Make sure we don't include content that only consists of
+                # numbers. See issue #13055.
+                if html_string != '' and not html_string.isnumeric():
```

Comment:
@sagangwee Considering the html for `100` can be `<p>100</p>`, is the `html_string.isnumeric()` helping in anyway with avoiding numeric translations?

Also, why don't we do it for the Unicode below? (and ditto for any content in general)

Reply:
Yes, I think you're right, we need to account for HTML tag case. There's no particular reason why we can't also do this for Unicode translations. Reopened the issue.